components/user_settings/notifications/notification_schedule_setting.scss

```diff
@@ -0,0 +1,194 @@
+            box-shadow: 0 0 1px #1b70ff;
+        }
+
+        input:checked + .slider::before {
+            -webkit-transform: translateX(20px);
+            -ms-transform: translateX(20px);
+            transform: translateX(20px);
+        }
+
+        /* Rounded sliders */
+        .slider.round {
+            border-radius: 50px;
+        }
+
+        .slider.round::before {
+            border-radius: 50%;
+        }
+    }
+}
+
+.form-select {
+    border: none;
+    margin: 0.8rem 0 0.8rem;
+    color: inherit;
+    font-size: inherit;
+    font-weight: 600;
+
+    .form-time {
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+        margin: 12px 0 8px 0;
+
+        .custom-time {
+            width: 100%;
+            margin: 4px 0 4px 0;
+
+            .week-name {
+                width: 100px;
+            }
+
+            .time-picker {
+                width: 150px;
+            }
+        }
+    }
+
+    div {
+        .period {
+            display: inline-block;
+            width: 35%;
+            margin-right: 10px;
+
+            .react-select__control {
+                width: 100%;
+
+                .react-select__value-container {
+                    width: 80%;
+                }
+
+                .react-select__indicators {
+                    width: 20%;
+                }
+            }
+        }
+
+        .n-custom-time {
+            width: 100%;
+
+            .time-picker {
+                width: 150px;
+            }
+        }
+
+        .time-wrapper {
+            display: inline-flex;
+            width: 60%;
+            align-items: center;
+
+            p {
+                margin: 0;
+            }
+
+            .time-picker {
+                display: flex;
+                align-items: center;
+                margin: 0 10px;
+
+                img {
+                    position: absolute;
+                    left: 8px;
+                    width: 24px;
+                    height: 24px;
+                    opacity: 0.5;
+                }
+
+                .rc-time-picker-input {
+                    width: 100%;
+                    height: 36px;
+                    padding-left: 38px;
+                    font-size: 14px;
+                    font-weight: 400;
+                }
+
+                .rc-time-picker-clear {
+                    display: none;
+                    width: 0;
+                    height: 0;
+                }
+            }
+        }
+    }
+}
+
+.weekDays-selector {
+    display: inline-block;
+
+    input {
+        display: none !important;
+    }
+
+    input[type=checkbox] + label {
+        display: inline-block;
+        width: 35px;
+        height: 35px;
+        margin-right: 8px;
+        background: #ddd;
+        border-radius: 20px;
+        cursor: pointer;
+        line-height: 35px;
+        text-align: center;
+    }
+
+    input[type=checkbox]:checked + label {
+        background: #1b70ff;
```

Comment:
please use `background: var(--button-bg);` to ensure it uses the appropriate theme color